platforms/osx/src/main.mm

```diff
@@ -13,11 +13,9 @@
 
 std::string sceneFile = "scene.yaml";
 
-std::string markerStyling = "{ style: 'points', interactive: true, color: 'white', size: [25px, 25px], order: 100, collide: false }";
+std::string markerStyling = "touch.point.icons";
```

Comment:
I commented on this earlier, but it was part of another comment so might not have been clear. This "path" is probably not going to make sense to a user. Even I, knowing the implementation, wouldn't find it intuitive.

The full path to the element we are trying to identify (i.e. the path we would use for a scene update) is `layers.touch.point.draw.icons`. If we are calling this a "path" then there's no reason to expect that certain parts should just be omitted.